scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -251,19 +279,20 @@ object Measurer {
       }
 
       var best = results
-      while (outlierExists(results) && retleft > 0) {
-        val prefixlen = suffixLength(results.reverse)
-        val suffixlen = suffixLength(results)
-        val formatted = results.map(t => f"$t%.3f")
+      while (outlierExists(results.map(_.value.toDouble())) && retleft > 0) {
+        val prefixlen = suffixLength(results.reverse.map(_.value.toDouble()))
+        val suffixlen = suffixLength(results.map(_.value.toDouble()))
+        val formatted = results.map(t => f"${t.value.toDouble()}%.3f")
```

Comment:
But will it be readable if we convert `Double` to `Double`? Won't the output in case of a lot of decimals look the same?

But, I guess this is not a big issue - I'm fine with either solution.